src/model/Deployment.ts

```diff
@@ -0,0 +1,29 @@
+import {IsNotEmpty, IsOptional} from 'class-validator';
+import {IsFridgeISO8601WithoutMs} from '../custom-validation-decorator/FridgeCustomValidationDecorators';
+
+export class Deployment {
+  @IsNotEmpty()
+  teamName: string;
+
+  @IsNotEmpty()
+  releaseTag: string;
+
+  @IsNotEmpty()
+  environment: string;
+
+  @IsNotEmpty()
+  applicationName: string;
+
+  @IsNotEmpty()
+  @IsFridgeISO8601WithoutMs()
+  timestamp: string;
+
+  @IsOptional()
+  cloudProvider: string;
+
+  @IsOptional()
+  region?: string;
```

Comment:
What do you think about making it required? Because if we force region to be required it will be easier for frontend to render data.
P.S. If there is no region in some team they can send "default" or "central"

Reply:
Will make it required then